viewer/viewer-awt/src/main/java/org/icepdf/ri/common/preferences/ExImportPreferencesPanel.java

```diff
@@ -0,0 +1,87 @@
+package org.icepdf.ri.common.preferences;
+
+import org.icepdf.ri.common.views.Controller;
+import org.icepdf.ri.util.ViewerPropertiesManager;
+
+import javax.swing.*;
+import javax.swing.border.EtchedBorder;
+import javax.swing.border.TitledBorder;
+import java.awt.*;
+import java.io.File;
+import java.util.ResourceBundle;
+
+public class ExImportPreferencesPanel extends JPanel {
+
+    private final GridBagConstraints constraints;
+
+    public ExImportPreferencesPanel(Controller controller, ViewerPropertiesManager propertiesManager, ResourceBundle messageBundle, Dialog parent) {
+        super(new GridBagLayout());
+        setAlignmentY(JPanel.TOP_ALIGNMENT);
+        final JButton exportButton = new JButton(messageBundle.getString("viewer.dialog.viewerPreferences.section.eximport.export.button.label"));
+        final JButton importButton = new JButton(messageBundle.getString("viewer.dialog.viewerPreferences.section.eximport.import.button.label"));
+
+        final JPanel panel = new JPanel(new GridBagLayout());
+        panel.setAlignmentY(JPanel.TOP_ALIGNMENT);
+        panel.setBorder(new TitledBorder(new EtchedBorder(EtchedBorder.LOWERED),
+                messageBundle.getString("viewer.dialog.viewerPreferences.section.eximport.border.label"),
+                TitledBorder.LEFT,
+                TitledBorder.DEFAULT_POSITION));
+
+        constraints = new GridBagConstraints();
+        constraints.fill = GridBagConstraints.NONE;
+        constraints.weightx = 1;
+        constraints.weighty = 0;
+        constraints.anchor = GridBagConstraints.CENTER;
+        constraints.insets = new Insets(5, 5, 5, 5);
+
+        exportButton.addActionListener(actionEvent -> {
+            final FileDialog chooser = new FileDialog(parent);
+            chooser.setMultipleMode(false);
+            chooser.setMode(FileDialog.SAVE);
+            chooser.setVisible(true);
+            final String dir = chooser.getDirectory();
+            final String file = chooser.getFile();
+            if (dir != null && file != null) {
+                if (ViewerPropertiesManager.exportProperties(new File(dir + file))) {
```

Comment:
PR, looks good. But can I get you to set the max line width to 120.  The showMessageDialogs really go on, doesn't have to be exactly 120 but somewhere in that ballpark.  maybe something more in the lines of :
JOptionPane.showMessageDialog(this, 
        messageBundle.getString("viewer.dialog.viewerPreferences.section.eximport.export.success.label"), 
        messageBundle.getString("viewer.dialog.viewerPreferences.section.eximport.export.success.title"), 
        JOptionPane.INFORMATION_MESSAGE);